src/hash.rs

```diff
@@ -0,0 +1,13 @@
+//! Hashing utilities.
+
+use std::hash::{Hash, Hasher};
+use twox_hash::Xxh3Hash64;
+
+const SEED: u64 = 0x7f95_ef85_be33_c337u64;
+
+/// Default hashing function used to shard records across workers.
+pub fn default_hash<T: Hash>(x: &T) -> u64 {
```

Comment:
What are the rules about the hash function? Do users need to know about this? Can they get a program wrong by somehow mixing multiple different hash functions? If so this has to be documented somewhere.

Reply:
I don't think there are any special requirements.